editor/questionbankcontroller.js

```diff
@@ -172,6 +171,19 @@ angular.module('createquestionapp', [])
         }
       });
     };
+    //Load all plugins
+    $scope.loadPlugins = function(plugins, manifestMedia) {
+      var pluginObj = [];
+      if (!Array.isArray(plugins)) {
+        pluginObj.push(plugins);
+        plugins = pluginObj;
+      }
+      org.ekstep.pluginframework.pluginManager.loadAllPlugins(plugins, manifestMedia, function() {
+        if (typeof PluginManager != 'undefined') {
+          PluginManager.pluginMap = org.ekstep.pluginframework.pluginManager.plugins;
```

Comment:
Why this is implemented? I think this has already taken care by plugin-framework.

Reply:
Question unit (MCQ/MTF/FTB)plugins not loading by plugin framework, So we are getting QS related units plugins by services and loading implicitly in the Question bank.